docs/man/git-lfs-migrate.1.ronn

```diff
@@ -271,7 +272,8 @@ $ git lfs migrate import --no-rewrite test.zip *.mp3 *.psd
 With a specified commit message:
 
 ```
-$ git lfs migrate import --no-rewrite -m "Import .zip, .mp3, .psd files" \
+$ git lfs migrate import --no-rewrite \
+  -m "Import .zip, .mp3, .psd files in root of repo" \
```

Comment:
Just getting confused with man book(`git-lfs/2.6.1 (GitHub; linux amd64; go 1.11.3)`) and found this PR.

According to the actual arguments(`test.zip *.mpd *.psd`) passed to git-lfs-migrate, to be more accurate, this commit message should be:

```diff
--  -m "Import .zip, .mp3, .psd files in root of repo" \
++  -m "Import test.zip, .mpd, .psd files in root of repo" \
```

And the [previous example](https://github.com/git-lfs/git-lfs/pull/3424/files#diff-4e71818f2c658bd6d213d1be9012df10R269) is talking about `test.zip *.mp3 *.psd`, I think this example should be:

```diff
 $ git lfs migrate import --no-rewrite \
--  -m "Import .zip, .mp3, .psd files in root of repo" \
++  -m "Import test.zip, .mp3, .psd files in root of repo" \
--   test.zip *.mpd *.psd
++   test.zip *.mp3 *.psd
```

Reply:
I've gone ahead and made that change.